ui-ux-design/ui-ux-design.md

```diff
@@ -89,11 +89,10 @@ Find a pair and share your findings for Activity #1 and Activity #2. Swap pairs
 
 ### Check for Understanding
 
-With a partner, discuss:
-- What is UX?
-- What is UI?  
+Form a small group and discuss
+
+- What is UX and UI ?  
 - How are they different?
-- Cite an example of good UI.
```

Comment:
Can you put this back the way it was?  I want them to think about the difference between UI and UX.